sql/moz-fx-data-shared-prod/stripe_derived/customers_v1/query.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+  -- limit fields in stripe_derived so as not to expose sensitive data
+  created,
+  id,
+  ARRAY(
+    SELECT
+      IF(key = "userid", STRUCT("fxa_uid" AS key, TO_HEX(SHA256(value)) AS value), (key, value))
+    FROM
+      UNNEST(metadata)
+  ) AS metadata
+FROM
+  stripe_external.customers_v1
```

Comment:
@jklukas does this look right for hashing the fxa user id when it reaches a dataset accessible in redash?

Reply:
it's technically possible to hash it sooner, but making it hashed all the way back to where it first appears in `stripe_external.events_v1` would require hashing it in the python script before load, which feels like the wrong place to be applying this transformation.

Reply:
fixed